packages/components/input-number/src/input-number.vue

```diff
@@ -218,16 +226,16 @@ const setCurrentValue = (value: number | string | null | undefined) => {
   const newVal = verifyValue(value)
   if (oldVal === newVal) return
   data.userInput = null
-  emit('update:modelValue', newVal!)
-  emit('input', newVal)
```

Comment:
It may causes breaking change if we remove `input` event

Reply:
This results in redundant input event being fired.

https://user-images.githubusercontent.com/23100055/191425549-d77f45df-4291-4a86-a019-1634bdd5295d.mov